Various small documentation tweaks

gregrs-uk opened the pull request:
* Added full stops and capital letters for consistency.
* Fixed some typos and grammatical/punctuation errors.
* Added backticks for `code formatting`.

hadley commented on R/geom-point.r:
```diff
@@ -12,7 +12,7 @@
 #'
 #' @section Overplotting:
-#' The biggest potential problem with a scatterplot is overplotting: whenever
+#' The biggest potential problem with a scatterplot is overplotting; whenever
```
`:` seems correct to me here

hadley replied:
In fact, can you revert all your colon changes?

gregrs-uk commented on R/geom-path.r:
```diff
@@ -6,16 +6,16 @@
 #' connected together.
 #'
-#' An alternative parameterisation is [geom_segment()]: each line
+#' An alternative parameterisation is [geom_segment()]; each line
```
I agree. Done.

gregrs-uk commented:
Thanks. I've reverted cases where I had changed a colon to a semicolon.

I've left the change to `R/aes-colour-fill-alpha.r` line 4 (sorry, not sure how to provide a link) because I believe the previous semicolon was incorrect.

gregrs-uk commented:
I tried to resolve the apparently conflicting files but it doesn't seem to have worked, apologies.

hadley commented:
This is lovely, thanks for all your hard work on this!

gregrs-uk commented:
No problem, thanks!
